### PR TITLE
Add simple PDF generation tool

### PR DESCRIPTION
**Pull request by ReeceHumphreys**

### CODE REVIEW

File: package.json
Comment (externl) [inline package.json:1]: We shouldn't include these dependencies in this package.json file.

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # misc
 .DS_Store
 *.pem
+output
```
Comment (externl) [inline .gitignore:19]: Is this necessary?

File: .vscode/cspell.json
Comment (externl) [inline .vscode/cspell.json:1]: I would inline these into one maybe two they're used it. `lillallol` probably shouldn't be a valid word :)

File: .vscode/cspell.json [hunk 2]
```diff
@@ -91,4 +94,4 @@
     "package.json",
     "public"
   ]
-}
+}
```
Comment (externl) [inline .vscode/cspell.json:97]: File should end in a newline.

File: tools/pdf/.gitignore
Comment (externl) [inline tools/pdf/.gitignore:1]: Many of the entires in this file are covered by the top level .gitignore.

File: tools/pdf/tsconfig.pdf.json
Comment (externl) [inline tools/pdf/tsconfig.pdf.json:1]: Missing final newline

File: tsconfig.json
Comment (externl) [inline tsconfig.json:1]: missing final newline

**Review by InsertCreativityHere — APPROVED** [review]
Looks good!
I'll just assume the code is all fine : v)

File: tools/pdf/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "pdf",
+  "version": "1.0.0",
+  "description": "Generate a PDF copy of the IceRPC docs",
+  "scripts": {
+    "generate-pdf": "ts-node -P tsconfig.pdf.json index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (externl) [inline tools/pdf/package.json:11]: This should probably use the same license as the rest of the repo.

EDIT: I would probably just set it private and remove all these fields like the top level one.

**Review by externl — APPROVED** [review]

**Pull request merged**